docs/intro/datamodel.md

```diff
@@ -45,7 +45,7 @@ You can set a custom primary key field using the `@db.primary` field annotation.
 type Note {
   id: ID!
   """
-  @db.primary
+  @db.id
```

Comment:
Thoughts on just `@id`?

I'd rather:

- `@unique` over `@db.unique`
- `@oneToMany` over `@db.oneToMany`

etc. where possible because `@db` does not add much information to above examples.

It comes from graphql-migrations and we could find more approiate namespaces like:

`@col(type: 'varchar', length: 200)` instead of `@db(type: 'varchar', length: 200)`

Reply:
I honestly thought that everything was under `@db` hence I went for consistency. Though I agree that `@db` does not add much value, there is probably a historical technical reason on top of it coming from graphql migrations which I would like to know :-)

Reply:
> How does this affect graphql-migrations?

Yeap, migrations was not handled. Good spot, I can add it too but let's decide ourselves first if we want `@db.id` or `@id`.

Reply:
graphql-migrations is a fork of [graphql-migrate](https://github.com/Akryum/graphql-migrate), which used `@db` everywhere.

My guess is because it was a tool for one thing - db migrations.

Graphback ecosystem uses these annotations in several context - for migrations, at runtime in CRUD services and data providers, and for schema generation.

Reply:
Thanks.

Reply:
So should we stick with `@db.id` or start the work of removing the `@db` prefix here? - meaning keeping this PR as simple as possible and do for `id` and followup PR for the rest of the remaining annotations.

Reply:
We can just do `@id` here, and do the other annotations in a separate sprint (we will need to migrate to a new format anyway, meaning a breaking change for these but not for `@id` as format wont change.

In fact you can use `parseMetadata` for `@id` now, it should accept it.